tests/templates/test_layers/test_particle_conserving_u1.py

```diff
@@ -119,10 +119,12 @@ def test_operations(self):
                     if exp_gate is qml.CRot:
                         if j < idx_CRot:
                             exp_params = [-phi, np.pi, phi]
-                        if j > idx_CRot:
+                        elif j > idx_CRot:
                             exp_params = [phi, np.pi, -phi]
-                        if j == idx_CRot:
+                        elif j == idx_CRot:
                             exp_params = [0, 2 * theta, 0]
+                        else:
+                            assert False, "Impossible index"
```

Comment:
I mean, we don't need to check Python's integer comparing capabilities, do we? :sweat_smile: 
```suggestion
                        else:
                            exp_params = [0, 2 * theta, 0]
```

Reply:
Lol true 🤦🏼